setup.py

```diff
@@ -13,7 +13,7 @@
 ]
 INSTALL_REQUIRES = [
     'numpy<=1.22.3', 'pandas', 'tqdm', 'scipy', 'jax>=0.2.22', 'numpyro',
-    'pyarrow', 'matplotlib <= 3.6.0', 'hitran-api', 'bs4', 'vaex>=4.9.2',
+    'pyarrow', 'matplotlib <= 3.6.0', 'hitran-api', 'bs4', 'vaex', 'vaex-hdf5',
```

Comment:
Also see recent remark on Numpy constraint which might not be required anymore https://github.com/radis/radis/issues/548

Reply:
using pip, constraint in setup.py of radis prevents dependency check. Should we update setup.py in radis first?

```
Installed /opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/PeakUtils-1.3.4-py3.8.egg
error: numpy 1.24.2 is installed but numpy<=1.22.3 is required by {'radis'}
Error: Process completed with exit code 1.
```